styled-icons.js.org

jacobwgillespie opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

---

I am moving hosting of https://styled-icons.js.org from Netlify to Zeit Now.  In order to complete this move, I would need the following DNS TXT record added as well:

```
Type   Host                       Value
TXT    _now.styled-icons.js.org   QmdcYQYf9r5qKwvTTMTQBsHTWg8JqdbHcNJm97SPFyamAT
```

jacobwgillespie commented:
> can't verify site contents

The Zeit link for what this should look like after the change is https://styled-icons.now.sh.

It should match the current site (http://styled-icons.netlify.com/).

indus commented:
I've changed the record in JS.ORGs zonefile. There propably has to be some configuration made at the host. Let me know when things work so I can merge.

jacobwgillespie commented:
> I've changed the record in JS.ORGs zonefile.

Did you add the `TXT` record mentioned above as well?  I'm not able to retrieve the TXT validation record via `dig`.

```
Type   Host                       Value
TXT    _now.styled-icons.js.org   QmdcYQYf9r5qKwvTTMTQBsHTWg8JqdbHcNJm97SPFyamAT
```

indus commented:
Sorry - you are right. I missed to add the TXT. I just added it now.

jacobwgillespie commented:
> I just added it now.

Thank you!  It passed validation, but it's stuck in a loop trying to issue a SSL certificate.  I believe it will need the Cloudflare proxy disabled, so I've added `// noCF` to this PR.  Can you disable the Cloudflare proxy for this entry?

indus commented:
@jacobwgillespie the proxy was disabled.

jacobwgillespie commented:
It works, thanks!  I believe this PR is ready to merge.

indus commented:
Welcome https://styled-icons.js.org
